client/src/components/DetailsPage/index.js

```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+import imgMeal from '../../assets/shrimppizza.jpg';
+import Button from '../utils/Button';
+import './style.css';
+
+const Details = () => {
```

Comment:
Use the props coming from the route in order to determine what meal you are trying to show, and send a request to the server to fetch the data associated with that meal